src/metrics/representation.py

```diff
@@ -320,8 +324,8 @@ def generator():
  
                 if self.metric_distance == MetricDistance.IP:
                     closest_dist = 1 - closest_dist
-            
-                closest = map(lambda idx: np.stack((db_ids[closest_idx[idx]], closest_dist[idx]), axis=1), \
+
+                closest = map(lambda idx: np.stack((db_ids[closest_idx[idx]], [1] * len(closest_idx[idx])), axis=1), \
```

Comment:
Why distances is always 1?

Reply:
it's dosen't important, for NDCG score=distance need for sort more relevant examples, but in this part of code we had already sorted our examples by faiss. So if we change 1 o distance with type float the index of relevant will be also float and after that inside ranx it may be fail to compare relevant int index with our relevant float index